services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/modules/db/repositories/resource_tracker.py

```diff
@@ -106,7 +105,7 @@ async def list_containers_by_user_and_product(
 
             result = await conn.execute(query)
             containers_list = [
-                ContainerGetDB.construct(**row)  # type: ignore[arg-type]
+                ContainerGetDB(**row)  # type: ignore[arg-type]
```

Comment:
why now validating and not before?

Reply:
How is this change of column names going to affect the data already in place?
Have you considered using `alias` in `ConainerGetDB` to account for this?

Reply:
- I have not validated it before, because I used just types that were 1-2-1 with the Pydantic model, now I introduced for example SQL decimal type: Decimal('3.50') -> but the pydantic model has a float so it is parsed/validated for me, but I can do it manually and remove the validation step if needed. 
- As we are in the development phase and data are just in the master (and not updated for a couple of days as the labels were changed) I have deleted them in the migration script.